.github/workflows/test.yml

```diff
@@ -150,10 +151,12 @@ jobs:
           echo "CONTAINER_ID=$CONTAINER_ID"  >> $GITHUB_ENV
       - name: Build lmdeploy from source
         run: |
-          docker exec $CONTAINER_ID cp -r ../../build build
+          docker exec $CONTAINER_ID mkdir build
```

Comment:
test_triton failed. It will be fixed in another PR.

Reply:
rollback down